packages/node/src/network-configuration.ts

```diff
@@ -40,26 +40,13 @@ interface Migration {
 function getContractAddressesForNetwork(
   migrations: Migration[]
 ): NetworkContext {
-  return {
-    AppRegistry: getContractAddress(migrations, "AppRegistry"),
-    ETHBalanceRefundApp: getContractAddress(migrations, "ETHBalanceRefundApp"),
-    ETHBucket: getContractAddress(migrations, "ETHBucket"),
-    MultiSend: getContractAddress(migrations, "MultiSend"),
-    NonceRegistry: getContractAddress(migrations, "NonceRegistry"),
-    StateChannelTransaction: getContractAddress(
-      migrations,
-      "StateChannelTransaction"
-    ),
-    ETHVirtualAppAgreement: getContractAddress(
-      migrations,
-      "ETHVirtualAppAgreement"
-    ),
-    MinimumViableMultisig: getContractAddress(
-      migrations,
-      "MinimumViableMultisig"
-    ),
-    ProxyFactory: getContractAddress(migrations, "ProxyFactory")
-  };
+  const ret = {} as any;
```

Comment:
Why `any` if the expected type is `NetworkContext`?

Reply:
`{}` does not conform to the type `NetworkContext`

Reply:
`tsc` doesn't error on that, but carry on. Just generally avoidant of using `any`